synapseclient/client.py

```diff
@@ -739,6 +739,12 @@ def _getWithEntityBundle(self, entityBundle, entity=None, **kwargs):
         ifcollision = kwargs.get('ifcollision', 'keep.both')
         submission = kwargs.get('submission', None)
 
+        #If Link, get target ID entity bundle
+        if entityBundle['entity']['concreteType'] == 'org.sagebionetworks.repo.model.Link':
+            targetId = entityBundle['entity']['linksTo']['targetId']
+            targetVersion = entityBundle['entity']['linksTo']['targetVersionNumber']
+            entityBundle = self._getEntityBundle(targetId, targetVersion)
```

Comment:
If you add a link to a file, shouldn't the annotations should be the same as that of the file its being linked to?  So the meta information about a link is just the meta information of the target file.